Quaver.Shared/Screens/Result/UI/ResultHitDifferenceGraph.cs

```diff
@@ -0,0 +1,305 @@
+                // {
+                //     Processor.Stats.Add(new HitStat(HitStatType.Hit, KeyPressType.Press, null, i * 2, Judgement.Good, i * 5, 0, 0));
+                //     Processor.Stats.Add(new HitStat(HitStatType.Hit, KeyPressType.Press, null, i * 2 + 1, Judgement.Good, -i * 5, 0, 0));
+                // }
+            }
+
+            FilterHitStats();
+            CreateDots(ConvertDifferenceToY);
+        }
+
+        /// <summary>
+        /// Returns the score processor to use. Loads hit stats from a replay if needed.
+        /// </summary>
+        /// <param name="screen"></param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentOutOfRangeException"></exception>
+        private static ScoreProcessor GetScoreProcessor(ResultScreen screen)
+        {
+            // If we already have stats (for example, this is a result screen right after a player finished playing a map), use them.
+            if (screen.ScoreProcessor.Stats != null)
+            {
+                return screen.ScoreProcessor;
+            }
+
+            // Otherwise, get the stats from a replay.
+            Replay replay = null;
+
+            // FIXME: unify this logic with watching a replay from a ResultScreen.
+            try
+            {
+                switch (screen.ResultsType)
+                {
+                    case ResultScreenType.Gameplay:
+                    case ResultScreenType.Replay:
+                        replay = screen.Replay;
+                        break;
+                    case ResultScreenType.Score:
+                        // Don't do anything for online replays since they aren't downloaded yet.
+                        if (!screen.Score.IsOnline)
+                            replay = new Replay($"{ConfigManager.DataDirectory.Value}/r/{screen.Score.Id}.qr");
+                        break;
+                    default:
+                        throw new ArgumentOutOfRangeException();
+                }
+            }
+            catch (Exception e)
+            {
+                NotificationManager.Show(NotificationLevel.Error, "Unable to read replay file");
+                Logger.Error(e, LogType.Runtime);
+            }
+
+            // Load a replay if we got one.
+            if (replay != null)
+            {
+                var qua = ResultScreen.Map.LoadQua();
+                MapLoadingScreen.ApplyModsToQua(qua, replay.Mods);
+                var player = new VirtualReplayPlayer(replay, qua);
+                player.PlayAllFrames();
+                return player.ScoreProcessor;
+            }
+
+            // This has no stats, return just as a fallback.
+            return screen.ScoreProcessor;
+        }
+
+        /// <summary>
+        /// Creates the middle line at 0 ms hit difference.
+        /// </summary>
+        /// <param name="middle"></param>
+        private void CreateMiddleLine(int middle)
+        {
+            var _ = new Sprite()
```

Comment:
You can use a lambda for this since it's just one line. Disable the resharper warning if you're using it as well.

**Suggested Change**

```cs
        /// <summary>
        /// Creates the middle line at 0 ms hit difference.
        /// </summary>
       // ReSharper disable once ObjectCreationAsStatement
        private void CreateMiddleLine() => new Sprite
        {
            Parent = this,
            Alpha = 0.7f,
            Alignment = Alignment.MidCenter,
            Size = new ScalableVector2(Width, 1),
        };
```